parking_lot_src/ParkingLot/ParkingLot.java

```diff
@@ -0,0 +1,104 @@
+package ParkingLot;
```

Comment:
This class looks like repository candidate, this is the place where you are holding data and doing some operation relation to data